donate.html

```diff
@@ -216,6 +216,7 @@ <h4 class="ak-donate-area-step-1">Donation (Optional)</h4>
                           $){% endfilter %}
                           {% elif currency == "JPY" %} {% filter ak_text:"general_ja_yen" %}Japanese Yen (JPY ¥){% endfilter %}
                           {% elif currency == "NZD" %} {% filter ak_text:"general_nz_dollar" %}New Zealand Dollar (NZD $){% endfilter %}
+                          {% elif currency == "ZAR" %} {% filter ak_text:"general_nz_dollar" %}South African Rand (ZAR R){% endfilter %}

```

Comment:
do we need to update the translation id from "general_nz_dollar"? (sorry, I forget a bit how this operates...) Does 350 need to add translation strings to the actionkit languages config for ZAR that is then referenced here?

Reply:
That will be updated in ActionKit!